.changeset/early-jobs-fail.md

```diff
@@ -0,0 +1,50 @@
+---
+'@shopify/hydrogen': patch
+---
+
+Adopt `v2_routeConvention` future flag
+
+### `v2_routeConventions` migration steps
+
+Remix v2 route conventions are just file renames. We just need to ensure when changing file name and file location, the import paths of other files are also updated.
+
+Here is an example list of file renames for the demo store:
```

Comment:
This is a confusing changeset for a developer to read and understand. Can you clarify what we expect developers to do in this release? Is this a breaking change for devs using Hydrogen?